### PR TITLE
add disco_diffusion_clip_rn101 module

### PR DESCRIPTION
**Pull request by rainyfly**
This module is developed based on [discoart](https://github.com/jina-ai/discoart), which uses guided denoising diffussion model to generate images semantically close to input text prompts. Here, RN101 model is used to guide generation process.

### CODE REVIEW

File: modules/image/text_to_image/disco_diffusion_clip_rn101/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# disco_diffusion_clip_rn101
+
+|模型名称|disco_diffusion_clip_rn101|
+| :--- | :---: |
+|类别|图像-文图生成|
+|网络|dd+clip RN101|
```
Comment (nepeplwu) [inline modules/image/text_to_image/disco_diffusion_clip_rn101/README.md]:
```suggestion
|网络|dd+clip ResNet101|
```

File: modules/image/text_to_image/disco_diffusion_clip_rn101/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  <br />
+
+  - 生成过程
+  <p align="center">
+    <img src="https://user-images.githubusercontent.com/22424850/184836085-beef3562-f89d-405e-ae04-2d3cd8499df8.gif"  width = "80%" hspace='10'/>
+  <br />
+
+### 模型介绍
+
+disco_diffusion_clip_rn101 是一个文图生成模型，可以通过输入一段文字来生成符合该句子语义的图像。该模型由两部分组成，一部分是扩散模型，是一种生成模型，可以从噪声输入中重建出原始图像。另一部分是多模态预训练模型（CLIP), 可以将文本和图像表示在同一个特征空间，相近语义的文本和图像在该特征空间里距离会更相近。在该文图生成模型中，扩散模型负责从初始噪声或者指定初始图像中来生成目标图像，CLIP负责引导生成图像的语义和输入的文本的语义尽可能接近，随着扩散模型在CLIP的引导下不断的迭代生成新图像，最终能够生成文本所描述内容的图像。该模块中使用的CLIP模型结构为RN101。
```
Comment (nepeplwu) [inline modules/image/text_to_image/disco_diffusion_clip_rn101/README.md]:
```suggestion
disco_diffusion_clip_rn101 是一个文图生成模型，可以通过输入一段文字来生成符合该句子语义的图像。该模型由两部分组成，一部分是扩散模型，是一种生成模型，可以从噪声输入中重建出原始图像。另一部分是多模态预训练模型（CLIP）, 可以将文本和图像表示在同一个特征空间，相近语义的文本和图像在该特征空间里距离会更相近。在该文图生成模型中，扩散模型负责从初始噪声或者指定初始图像中来生成目标图像，CLIP负责引导生成图像的语义和输入的文本的语义尽可能接近，随着扩散模型在CLIP的引导下不断的迭代生成新图像，最终能够生成文本所描述内容的图像。该模块中使用的CLIP模型结构为ResNet101。
```

Comment (rainyfly) [reply modules/image/text_to_image/disco_diffusion_clip_rn101/README.md]: Done

File: modules/image/text_to_image/disco_diffusion_clip_rn101/README.md (hunk 1 above)
Comment (rainyfly) [reply modules/image/text_to_image/disco_diffusion_clip_rn101/README.md]: Done

**Pull request merged**